spi/control-plane/policy-spi/src/testFixtures/java/org/eclipse/edc/connector/policy/spi/testfixtures/store/PolicyDefinitionStoreTestBase.java

```diff
@@ -606,6 +606,75 @@ void findAll_with_privateProperties() {
             assertThat(list).hasSize(2).usingRecursiveFieldByFieldElementComparator().isSubsetOf(policy1, policy2);
         }
 
+
+        @Test
+        void shouldReturn_with_private_propertiesFilter() {
```

Comment:
Again, verify the correct documents are returned in this method

Reply:
done